types/knex/index.d.ts

```diff
@@ -153,6 +153,7 @@ declare namespace Knex {
         pluck(column: string): QueryBuilder;
 
         insert(data: any, returning?: string | string[]): QueryBuilder;
+        modify(callback: QueryCallback): QueryBuilder;
```

Comment:
According to the linked docs, this should be `modify(callback: QueryCallback, ...args: any): QueryBuilder;`.

Reply:
Thx. Added.